x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/unavailable_prompt.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+import { EuiEmptyPrompt } from '@elastic/eui';
+
+export const UnavailablePrompt: React.FC = () => (
+  <EuiEmptyPrompt
+    iconType="clock"
+    title={
+      <h2>
+        {i18n.translate('xpack.enterpriseSearch.appSearch.engine.overview.unavailableTitle', {
+          defaultMessage: 'Dashboard metrics are currently unavailable',
```

Comment:
```suggestion
          defaultMessage: 'Dashboard metrics are currently unavailable.',
```

Reply:
👍 For a lot of these casing comments, I was copying these over from ent-search as-is, but I think it makes a lot of sense to update them as we migrate. Will do so going forward!

Reply:
I went back and looked at some of EuiEmptyPrompt examples and I think the headings are actually not supposed to have punctuation: https://elastic.github.io/eui/#/display/empty-prompt and

<img width="645" alt="Screen Shot 2020-11-18 at 10 55 35 AM" src="https://user-images.githubusercontent.com/549407/99574732-9c566500-298c-11eb-86c5-9d74cce42d8d.png">

So going to leave this one as-is 🤷

Reply:
Makes sense 👍